migrations/versions/22828222f2fe_add_bakery_table.py

```diff
@@ -0,0 +1,65 @@
+"""Add bakery table
+
+Revision ID: 22828222f2fe
+Revises: 81c7d27bde71
+Create Date: 2022-01-19 14:35:15.140807
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "22828222f2fe"
+down_revision = "81c7d27bde71"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "bakery",
+        sa.Column("region", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("description", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("id", sa.Integer(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_bakery_description"), "bakery", ["description"], unique=False)
+    op.create_index(op.f("ix_bakery_id"), "bakery", ["id"], unique=False)
+    op.create_index(op.f("ix_bakery_name"), "bakery", ["name"], unique=False)
+    op.create_index(op.f("ix_bakery_region"), "bakery", ["region"], unique=False)
+    op.create_index(op.f("ix_reciperun_bakery_id"), "reciperun", ["bakery_id"], unique=False)
+    op.create_index(op.f("ix_reciperun_completed_at"), "reciperun", ["completed_at"], unique=False)
+    op.create_index(op.f("ix_reciperun_conclusion"), "reciperun", ["conclusion"], unique=False)
+    op.create_index(op.f("ix_reciperun_feedstock_id"), "reciperun", ["feedstock_id"], unique=False)
+    op.create_index(op.f("ix_reciperun_head_sha"), "reciperun", ["head_sha"], unique=False)
+    op.create_index(op.f("ix_reciperun_id"), "reciperun", ["id"], unique=False)
+    op.create_index(op.f("ix_reciperun_message"), "reciperun", ["message"], unique=False)
+    op.create_index(op.f("ix_reciperun_recipe_id"), "reciperun", ["recipe_id"], unique=False)
+    op.create_index(op.f("ix_reciperun_started_at"), "reciperun", ["started_at"], unique=False)
+    op.create_index(op.f("ix_reciperun_status"), "reciperun", ["status"], unique=False)
+    op.create_index(op.f("ix_reciperun_version"), "reciperun", ["version"], unique=False)
```

Comment:
I don't understand why all these indexes are being created by this migration.

Reply:
This migration was deleted in https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/e64d7eae3ed3ff5f823359d8429aaa79597782cf, but the underlying question remains a mystery.

We are not defining any [SQLModel indexes](https://sqlmodel.tiangolo.com/tutorial/indexes/), so it's not that.

AFAICT, Alembic uses SQLAlchemy's [`get_indexes`](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#controlling-what-to-be-autogenerated) inspection to generate these `create_index` calls. Specifically, that's called as a [method of an `inspector` object in `alembic.autogenerate.compare`](https://github.com/sqlalchemy/alembic/blob/85152025ddba1dbeb51b467f40eb36b795d2ca37/alembic/autogenerate/compare.py#L558).

So it seems like these indexes must be inferred from some other attributes on that `inspector` object, or perhaps (more likely?) from the `schema` kwarg being passed to `get_indexes`. There's some type of semi-opaque decorator registration happening in Alembic there, so I didn't get to the point of actually tracking down where, e.g., that `schema` value is defined.

Reply:
This looks like https://github.com/tiangolo/sqlmodel/issues/9